edx_proctoring/backends/rest.py

```diff
@@ -184,7 +184,11 @@ def register_exam_attempt(self, exam, context):
         payload['status'] = 'created'
         # attempt code isn't needed in this API
         payload.pop('attempt_code', False)
-        log.debug(u'Creating exam attempt for %r at %r', exam['external_id'], url)
+        log.debug(
+            'Creating exam attempt for exam_id = {exam_id} (external_id = {external_id}) at {url}'.format(
```

Comment:
Something worth confirming is if spunk can extract the field with spaces on either side of equals. You may have to reformat this as `key=val`

If you already know the spaces work, disregard this.

Reply:
Good call, I think you're right